README.md

```diff
@@ -1,2 +1,21 @@
+![image](images/sunglasses.jpg)
+
 # awesome-conda-plugins
```

Comment:
Could we just rename this to `conda/plugins` to not imply that it'll follow the `awesome` maintenance pattern (solo reviewers usually)?

Context: We should be striving towards maintaining the metadata about existing packages that contain plugin implementations via a conda community subteam dedicated to that work, whose focus should *not* be deciding if a plugin is "awesome", as that would artificially set a barrier of entry, but just reviewing if the info provided by the maintainers is correct to prevent malicious behavior.

Setting up the criteria for selection and follow through with a `awesome` repo is much more involved and should be first discussed with the steering council.

Reply:
I've moved the repo to the incubator (via the regular incubation pattern in the conda governance policy) and added @kenodegard @travishathaway and you to the project team 🤗